dwave/system/samplers/clique.py

```diff
@@ -165,17 +165,16 @@ def sample(self, bqm, chain_strength=None, **kwargs):
 
         if chain_strength is None:
             # chain length determines chain strength
-            if embedding:
-                chain_strength = 1.5 * math.sqrt(len(embedding))
+            if embedding and bqm.num_variables > 1:
+                squared_j = (j ** 2 for j in bqm.quadratic.values())
+                num_edges = (bqm.num_variables*(bqm.num_variables-1))/2
```

Comment:
should this be `bqm.num_interactions`? Or do we want the maximum number of possible edges? Would apply for nearly-dense BQMs.

Reply:
i don't know the answer to this question 😮

Reply:
@pau557 ?

Reply:
`bqm.num_interactions` should be the correct choice yes

Reply:
to clarify, in case we pass a clique as a bqm, then there is no difference between that and `(bqm.num_variables*(bqm.num_variables-1))/2`, otherwise if we pass a sparser bqm to the clique sampler, the correct version is `bqm.num_interactions`.